airflow/www/templates/airflow/dag.html

```diff
@@ -151,7 +151,8 @@ <h4 class="pull-right js-dataset-triggered" style="user-select: none;-moz-user-s
         {% endif %}
         {% if dag_model is defined and dag_model.next_dagrun is defined and dag_model.schedule_interval != 'Dataset' %}
           <p class="label label-default js-tooltip" style="margin-left: 5px" id="next-run" data-html="true" data-placement="bottom">
-            Next Run ID: <time datetime="{{ dag_model.next_dagrun }}">{{ dag_model.next_dagrun }}</time>
+            Next Run ID: <time datetime="{{ dag_model.next_dagrun }}">{{ dag_model.next_dagrun }}
+              {{ dag_model.next_dagrun.tzinfo }}</time>
```

Comment:
@bbovenzi @pierrejeambrun With the fix added in the dag.html, I could see that time has timezone info (both in text `UTC` and numerical `+00:00` representation ) on loading, but then after loading it goes back to the old state. Am I missing something to notice?